### PR TITLE
Ac 29672 terraform add collection id to policy app

### PR DESCRIPTION
**Pull request by yuvalstyrbrod**
1. add ability to create policy or app with collection id 
2. assign roles: site, collection and tenant

### CODE REVIEW

**Comment by yiftahHaim** [comment]
please remove all policy entity assigner code

File: service/role_bindings.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		return nil, err
+	}
+	roles, _, err := r.cli.CollectionsApi.CreateTenantRoleBinding(context.Background(), *body)
+	if err != nil {
+		return nil, err
+	}
+	return dto.ConvertRolesBindingsToDTO(roles), nil
+}
+
+// CreateCollectionRoleBindings assign tenant role to admin
```
Comment (gitomics) [inline service/role_bindings.go:33]: assign tenant role to admin
->
assign collection role to admin

File: service/role_bindings.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	}
+
+	roles, _, err := r.cli.CollectionsApi.CreateCollectionRoleBinding(context.Background(), *body)
+	if err != nil {
+		return nil, err
+	}
+	return dto.ConvertRolesBindingsToDTO(roles), nil
+}
+
+// CreateSiteRoleBindings assign tenant role to admin
```
Comment (gitomics) [inline service/role_bindings.go:47]: assign site role to admin

File: service/role_bindings.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	}
+
+	roles, _, err := r.cli.CollectionsApi.CreateSiteRoleBinding(context.Background(), *body)
+	if err != nil {
+		return nil, err
+	}
+	return dto.ConvertRolesBindingsToDTO(roles), nil
+}
+
+// ReadRoleBindings get tenant role
```
Comment (gitomics) [inline service/role_bindings.go:61]: get tenant role
 ->
  get role

**Review by gitomics — CHANGES_REQUESTED** [review]

File: service/dto/dto.go [hunk 4]
```diff
@@ -144,7 +146,6 @@ const (
 	ManagedDeviceCondition                 = "IS_DEVICE_COMPLIANCE"
 	ManagedDeviceCloudSocConditionArgument = "CloudSOC"
 	ManagedDeviceOpswatConditionArgument   = "OPSWAT"
-	ManagedDeviceOpswatGroupsArgument      = "OPSWAT_GROUPS"
```
[the remark this answers is not in the document]
Comment (yuvalstyrbrod) [reply service/dto/dto.go:147]: not related to rbac, but, needed to pass all AT due to a change in policy-service

File: service/dto/access_policy_converter.go [hunk 5]
```diff
@@ -211,7 +200,6 @@ func ConvertToDto(accessPolicy *AccessPolicy) sdk.PolicyAccess {
 			if accessPolicy.Conditions.ManagedDevice.OpswatMetaAccess {
 				managedDeviceArguments = append(managedDeviceArguments, ManagedDeviceOpswatConditionArgument)
 			}
-			argumentsMap[ManagedDeviceOpswatGroupsArgument] = []string{}
```
[the remark this answers is not in the document]
Comment (yuvalstyrbrod) [reply service/dto/access_policy_converter.go:214]: not related to rbac, but, needed to pass all AT due to a change in policy-service

**Review by yiftahHaim — APPROVED** [review]
looks good to me.

**Review by gitomics — APPROVED** [review]

**Pull request merged**